ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobs.tsx

```diff
@@ -139,12 +142,12 @@ function ReportJobs({ reportId }: RunHistoryProps) {
                     </EmptyStateTemplate>
                 </Bullseye>
             )}
-            {isLoading && (
+            {isLoading && !reportSnapshots && (
```

Comment:
This prevents removing/replacing the whole table with a loading spinner when the data updates.

Reply:
I think this also only allows the spinner to be seen in the initial load. When we try to filter by report name, I don't see the spinner show up. I would expect it to show up in that case. Maybe that's fine for now. I think we might need to figure out a way that allows the spinner to show up for certain cases (ie. initial load/filtering/pagination updates) and not others (ie sorting/column updates). We'd have to figure out a good way to go about that though, which goes beyond this PR

Reply:
I agree, this is something we should tackle in a general way, as it happens pretty much everywhere in the app.

A while back UX suggested something like having a loading spinner/three dots over top of the currently displayed data, set to a low opacity. That way there would be indication that data was loading without the jarring full-replacement of the table each time.